<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Coffee</class>
 <widget class="QWidget" name="Coffee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кофе</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>631</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Изменить таблицу</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
